csunplugged/tests/resources/loaders/test_resource_loader.py

```diff
@@ -2,9 +2,9 @@
 from tests.resources.ResourcesTestDataGenerator import ResourcesTestDataGenerator
 from resources.models import Resource
 from resources.management.commands._ResourcesLoader import ResourcesLoader
-from utils.errors.CouldNotFindConfigFileError import CouldNotFindConfigFileError
+from utils.errors.CouldNotFindYAMLFileError import CouldNotFindYAMLFileError
```

Comment:
This was 'Config' instead of YAML to make it easier for authors to identify, however the file extension is `.yaml` and the filepath is displayed in the message, so I'm happy for it to be changed.